.github/workflows/release.yml

```diff
@@ -1,17 +1,26 @@
 name: "release"
 on:
   push:
-    tags:
-      - '*'
+    tags: # Will upload to luarocks.org
+      - "*"
+  pull_request: # Will test a local install without uploading to luarocks.org
+   paths:
+    - 'contrib/*.rockspec'
+    - .github/workflows/release.yml
+  
 jobs:
   luarocks-upload:
     runs-on: ubuntu-latest
     steps:
-      - uses: nvim-neorocks/luarocks-tag-release@v2.2.0
+      - uses: actions/checkout@v3
+      - uses: nvim-neorocks/luarocks-tag-release@v5
         env:
           LUAROCKS_API_KEY: ${{ secrets.LUAROCKS_API_KEY }}
         with:
+          name: nvim-treesitter
+          version: 0.1
```

Comment:
Sorry, I'm a bit late to the party.
Just noticed that the version is fixed here.
Is this left over from testing?